docs/gen2_Tutorial.md

```diff
@@ -434,19 +434,19 @@ To create a gen2 cluster with autoscaler, we need:
 
 - bundle pointing to the bundle/capi-cluster-autoscaler in the arlon repository.
 - dynamic profile that contains the above bundle.
-- a basecluster manifest which has the CAPI annotations for min and max nodes set ( as a part of `preparegit` or manually add it ).
+- a basecluster manifest(that makes use of MachineDeployment and not MachinePools) which has the CAPI annotations for min and max nodes set ( as a part of `preparegit` or manually add it ).
```

Comment:
Note: Jaya has a separate PR to rename base clusters to cluster templates. Unless we fix this later, perhaps you start can using "cluster template" in this PR immediately.

Reply:
Yeah I'll rename it here....Will reduce further efforts